docs/ops/matrix/Einsum_7.md

```diff
@@ -0,0 +1,228 @@
+## Einsum <a name="Einsum"></a> {#openvino_docs_ops_matrix_Einsum_7}
+
+**Versioned name**: *Einsum-7*
+
+**Category**: Matrix multiplication
+
+**Short description**: *Einsum* performs the Einstein summation convention on the operands.
+
+**Detailed description**: *Einsum* can represent many common multidimensional linear algebraic tensor operations: matrix multiplication;
+inner (or dot), outer and cross products; transpose; trace and diagonal extraction.
+Also, a single *Einsum* operation can express complex combination of these common linear algebraic tensor operations on multiple operands,
+for example, a dot product of a diagonal, extracted from a tensor with shape [5, 5], and 5D vector is performed by single Einsum operation.
+The Einstein summation convention on input tensors is defined by `equation`, which is a mandatory attribute of *Einsum* operation.
+The format of `equation` is described below.
+
+In explicit mode, the einsum `equation` has the output subscript separated from the input subscripts by `->`, and has the following format for `n` operands: 
+`<subscript for input1>, <subscript for input2>, ..., <subscript for inputn> -> <subscript for output>`.
+Each input subscript `<subscript for input1>` contains a sequence of labels (lowercase letters `['a',...,'z']`), where each label refers to a dimension of 
+the corresponsing operand. Labels do not need to appear in a subscript in alphabetical order. The subscript for a scalar input is empty.
+The input subscripts are separated with a comma `,`.
+The output subscript `<subscript for output>` is separated from the input subscripts by `->` and represents a sequence of labels
```

Comment:
nit: we already said that the output subscript is separated from the input by '->' (in the beginning of the paragraph), so probably here we may omit this and just write: "The output subscript `<subscript for output>` represents a sequence of labels..."

Reply:
Fixed.